Allow Code Component to have custom props and extra class styles (#9960)
withastro/astro: .changeset/calm-bags-deliver.md

---
"@astrojs/markdown-remark": minor
"astro": minor
---

> Added functionality for additional props within the code component.

Reviewer: ```suggestion
Allows passing any props to the `<Code />` component
```
